operator/controller/pom.xml

```diff
@@ -89,6 +89,12 @@
       <artifactId>lombok</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>io.strimzi</groupId>
```

Comment:
Ideally we should be able to not tight the setup to Strimzi but keep it more generic.